src/gdgem/srs/CalibrationFile.h

```diff
@@ -16,6 +16,12 @@ namespace Gem {
 
 /// \todo check whether packing is necessary, static assert assert?
 struct Calibration {
+  //Apple clang demands constructors for the struct
+  Calibration()= default;
```

Comment:
The clang error I get in another project (my analysis code), that uses the calibration file classes, 
https://github.com/ess-dmsc/vmm-hdf5-to-root
I do not understand it, as you said the other classes also do not have that. But since it is a small change, and actually correct, I would like to leave it.